critiquebrainz/db/test/statistics_test.py

```diff
@@ -162,11 +162,10 @@ def test_get_top_users_overall(self, cache_set, cache_get):
         # get list of top users
         top_users = db_statistics.get_top_users_overall()
         expected_key = b"top_users_overall_cb_statistics"
-        cache_get.assert_called_once_with(expected_key,
-                                          db_statistics._CACHE_NAMESPACE)
-        cache_set.assert_called_once_with(key=expected_key, val={"users": top_users},
-                                          namespace=db_statistics._CACHE_NAMESPACE,
-                                          time=db_statistics._DEFAULT_CACHE_EXPIRATION)
+        cache_get.assert_called_with(expected_key, db_statistics._CACHE_NAMESPACE)
```

Comment:
why does the namespace occur twice here?
https://github.com/metabrainz/critiquebrainz/blob/75ddff3569036bb7296097d0b61d90684581dd74/critiquebrainz/db/statistics.py#L238-L239
we use gen_key to make a key with "top_users_overall" + "cb_statistics", and then we do cache.get with a specific namespace anyway. Key will end up being "cb_statistics:top_users_overall_cb_statistics". Don't need the gen_key in statistics.py